src/shapes/SurfacePolyline.js

```diff
@@ -101,5 +101,10 @@ define([
             return SurfacePolyline.staticStateKey(this);
         };
 
+        // Internal. Polyline doesn't generate its own boundaries. See SurfaceShape.prototype.computeBoundaries.
+        SurfacePolyline.prototype.computeBoundaries = function(dc) {
+            return;
```

Comment:
Same comment for the unnecessary return in SurfacePolygon.prototype.computeBoundaries.